packages/components/pie-button/README.md

```diff
@@ -170,3 +170,9 @@ Under scripts `test:visual` replace the environment variable with the below:
 ```bash
 PERCY_TOKEN_PIE_BUTTON=abcde
 ```
+
+# Peer Dependencies
+
+Before using `@justeattakeaway/pie-button`, please make sure you have the following peer dependencies installed in your project:
```

Comment:
I would probably put this up near line 60, where it shows how to import the React version.

Could be that you just add it just after the code example, like:

> **Note:** When using the React version of the component, please make sure you also include React as a dependency in your project as well.